startup/20-detectors.py

```diff
@@ -0,0 +1,77 @@
+from ophyd import (
+    AreaDetector,
+    SingleTrigger,
+    StatsPlugin,
+    ROIPlugin,
+    TIFFPlugin,
+    EpicsSignalRO,
+)
+
+
+class StandardCam(SingleTrigger, AreaDetector):
+    # Statistics
+    stats1 = Cpt(StatsPlugin, "Stats1:")
+    stats2 = Cpt(StatsPlugin, "Stats2:")
+    stats3 = Cpt(StatsPlugin, "Stats3:")
+    stats4 = Cpt(StatsPlugin, "Stats4:")
+    stats5 = Cpt(StatsPlugin, "Stats5:")
+
+    # Region of Interest
+    roi1 = Cpt(ROIPlugin, "ROI1:")
+    roi2 = Cpt(ROIPlugin, "ROI2:")
+    roi3 = Cpt(ROIPlugin, "ROI3:")
+    roi4 = Cpt(ROIPlugin, "ROI4:")
+
+    # TIFF saving
+    tiff = Cpt(TIFFPlugin, "TIFF1:")
```

Comment:
Do you expect to save tiffs? If so, probably the `write_path_template` and `root` should be specified too.

Reply:
Hmm the plugin is available but for the time being I think that won't be needed. I will remove that line